sdk/rust/src/pvrecorder.rs

```diff
@@ -9,15 +9,28 @@
     specific language governing permissions and limitations under the License.
 */
 
+use lazy_static::lazy_static;
 use libc::c_char;
 use libloading::{Library, Symbol};
 use std::cmp::PartialEq;
-use std::convert::AsRef;
 use std::ffi::CStr;
-use std::path::{Path, PathBuf};
 use std::ptr::addr_of_mut;
 use std::sync::Arc;
 
+lazy_static! {
+    static ref PV_RECORDER_LIB: Result<Library, RecorderError> = {
+        unsafe {
+            match Library::new(pv_library_path()) {
+                Ok(symbol) => Ok(symbol),
+                Err(err) => Err(RecorderError::new(
+                    RecorderErrorStatus::LibraryLoadError,
+                    &format!("Failed to load pvrecorder dynamic library: {}", err),
+                )),
+            }
+        }
+    };
+}
+
```

Comment:
We should probably do this for the other SDKs too to be honest. It's safer and with some more work we could actually guarantee the lifetimes of the function symbols at compile time rather than by logic.

Reply:
Did it!

Reply:
You should add a card to tracking for this maybe if it's a significant improvement to the other Rust SDKs.

Reply:
:+1: